src/main/resources/db/migration/V1_3__seed_service_category.sql

```diff
@@ -0,0 +1,9 @@
+insert into service_category (id, name) values ('d21fc249-41ff-4987-8027-0c559f9bbd2e', 'Accommodation');
```

Comment:
the uuid for this category currently in use in the pact tests is `428ee70f-3001-4399-95a6-ad25eaaede16`, easiest to match it here.

Reply:
Definitely.  Updated